docs/docs/api/components/keyboard-toolbar/index.mdx

```diff
@@ -215,9 +215,10 @@ A boolean prop indicating whether to show `next` and `prev` buttons. Can be usef
 Prop allowing you to specify the brand colors of your application for `KeyboardToolbar` component. If you want to re-use already platform specific colors you can import `DefaultKeyboardToolbarTheme` object and override colors only necessary colors:
 
 ```ts
-import { DefaultKeyboardToolbarTheme } from "react-native-keyboard-controller";
+import { DefaultKeyboardToolbarTheme, type KeyboardToolbarProps } from "react-native-keyboard-controller";
```

Comment:
I didn't add `type` keyword in documentation (typically if people are using it, then they have eslint that automatically does correction) - let's be consistent across documentation and remove `type` from import statement 🙏